case_studies/san_leandro_high_school/02_first_meeting/planning.md

```diff
@@ -1 +1,14 @@
 # Planning
+
+### Plans for event
+1. at the entrance we will wave our logo wildly to make sure members see us, as we are located in a less known part of school.
+2. start off by giving the same speech given at [hack camp](https://github.com/hackedu/hack-camp/blob/master/cohort_4/playbook/ACTIVITIES.md#introduction-talk)
+3. give wifi password
+4. fill out survey, so we can invite them to slack
+5. send them on their way to create their own website with this [guide](https://github.com/hackedu/hack-camp/tree/master/cohort_4/playbook/workshops/portfolio) from hack camp.
+6. do an activity where we can all get to know each other.
+7. do the hack cheer!
+
+### anticipated problems
+* lack of directors to peer ratio, as 3/5 of our directors are doing sports the beginning 30min
+* technical difficulty with the projector system
```

Comment:
Can you add a new line after line 12 to conform to our Markdown guidelines? Also make sure that the entire file is wrapped at 80 characters.